NonsoAmadi10/Auto-Mart
# #166392573 Create endpoint  DELETE /api/v1/car/:id

### What does this PR do?
- Creates endpoint DELETE ` /api/v1/car/:id `

### Description of Task Taken To Complete??
- Setup Test Cases
- Implement this feature
- Run test against feature
- Protect route with JSON web token
- Validate routes

### How should this be manually Tested??
- Clone this repo
- Start server with ` npm run dev-start `
- Open Postman 
- Send a ` DELETE /api/v1/car/5` to see the implementation of this feature

###### Testing with Mocha
- Run npm test to see test results


### Any Background Context you want to provide??
- You must be an admin to use this feature. To register as an admin see #25 and pass in parameters ` firstname, lastname , email, password and is_admin set to 't' `
- Collect the token from the response and pass in to the authorization headers in postman and test this feature

### What are the relevant pivotal tracker stories??
[Delivers #166392573](https://www.pivotaltracker.com/story/show/166392573)
